src/control/server/storage/bdev/provider.go

```diff
@@ -139,16 +124,6 @@ func (p *Provider) Scan(req storage.BdevScanRequest) (resp *storage.BdevScanResp
 // Prepare attempts to perform all actions necessary to make NVMe
 // components available for use by DAOS.
 func (p *Provider) Prepare(req storage.BdevPrepareRequest) (*storage.BdevPrepareResponse, error) {
-	/*if p.shouldForward(req) {
```

Comment:
I probably missed an earlier discussion but why was this removed? It doesn't seem like it would interfere with your tiering related changes (setting a flag on the request seems orthogonal), I would have preferred to Mark the commented block with a TODO DAOS-XXX comment. This is not a blocking remark.

Reply:
Because the bdev/scm providers are only called from daos_admin now.